test.rb

```diff
@@ -0,0 +1,11 @@
+a = [1,2,3]
+puts a.find { |e| e.even? }
+puts a.find { |e| e.even? }
+puts a.find { |e| e.even? }
```

Comment:
Prefer `detect` over `find`.<br>Pass `&:even?` as an argument to `find` instead of a block.